### ISSUE TITLE
grunt website gave error

### ISSUE THREAD

**Issue by willscott** [freedomjs/freedom#183]
I'm not sure I understand how to even check what the expected behavior should be, but upon running ```grunt website``` for v0.6.14, I saw at the end of output:

```
>> err: Cloning into 'build/freedomjs'...
>> To git@github.com:freedomjs/freedomjs.github.io.git
>>    627ce12..a0f0a6f  master -> master
```

**Comment by ryscheng** [comment]
Yes, I get this behavior too.
This actually means that the 'grunt website' task has succeeded. You should see
```Done, without errors.``` correct?
We are using child_process to execute our publishWebsite.sh shell script, which for some reason outputs  the lines you mentioned above to stderr, not stdout. If you check freedomjs.github.io, it should contain all the files that it should have published. Not sure what to replace our existing setup with, but this is definitely a weird way to be logging.

**Comment by willscott** [comment]
Makes sense. This seems pretty low priority, and probably can be done with appropriate output piping / child_process arguments.
